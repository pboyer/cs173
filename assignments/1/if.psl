if false then print("then branch\n") else print("else branch\n");
if true then print("then branch\n") else print("else branch\n");
if {x: 5} then print("then branch\n") else print("else branch\n");
if 5 then print("then branch\n") else print("else branch\n");
if "true" then print("then branch\n") else print("else branch\n");
if "false" then print("then branch\n") else print("else branch\n");
if >(1,0) then print("then branch\n") else print("else branch\n");