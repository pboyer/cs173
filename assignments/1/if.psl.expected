else branch
then branch
then branch
then branch
then branch
then branch
then branch
then branch

